core/templates/dev/head/filtersSpec.js

```diff
@@ -410,4 +410,79 @@ describe('Testing filters', function() {
     expect(filter('  only First lettEr is  Affected ')).toEqual(
       'Only First lettEr is  Affected');
   }));
+
+  it('should correctly display RTE components', inject(function($filter) {
+    var ruleMath = {
+      type: 'Equals',
+      inputs: {
+        x: 2
+      }
+    };
+    var interactionIdMath = 'TextInput';
+    var choicesMath = [
+      {
+        label: '<oppia-noninteractive-math raw_latex-with-value="&amp;quot;' +
+        'x^3 - a x^2 - b x - c&amp;quot;"></oppia-noninteractive-math>',
```

Comment:
Indent by 2

Reply:
Not going to mark the rest of the file, but please indent the following in the same manner!